network/limitlistener/rejectingLimitListener_test.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2013 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package limitlistener_test
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"sync"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	"github.com/algorand/go-algorand/network/limitlistener"
+	"github.com/algorand/go-algorand/test/partitiontest"
+)
+
+const defaultMaxOpenFiles = 256
+const timeout = 5 * time.Second
+
+func TestRejectingLimitListener(t *testing.T) {
```

Comment:
```suggestion
func TestRejectingLimitListenerSoftLimit(t *testing.T) {
```

Reply:
why soft limit?

Reply:
Aren't the errors your checking for in this test the SoftLimit? These renames are just suggestions, if there is a better name than "Test*" and "Test*Error" that would be fine as well.

Reply:
There is no concept of soft and hard limit within this test, just the limit. Renamed it to`TestRejectingLimitListenerBasic`, not sure it makes a big difference.